services/submission/src/test/java/app/coronawarn/server/services/submission/controller/RequestExecutor.java

```diff
@@ -62,6 +62,10 @@ public RequestExecutor(TestRestTemplate testRestTemplate) {
 
   public ResponseEntity<Void> executePost(Collection<TemporaryExposureKey> keys, HttpHeaders headers) {
     SubmissionPayload body = SubmissionPayload.newBuilder().addAllKeys(keys).build();
+    return executePost(body, headers);
```

Comment:
Is the "old" method with the `Collection<TemporaryExposureKey> keys` parameter still needed?

Reply:
Good hint, refactored